analytics/build/build.go

```diff
@@ -130,3 +133,37 @@ func evaluateActivities(rw *openrtb_ext.RequestWrapper, ac privacy.ActivityContr
 	cloneReq.RebuildRequest()
 	return true, cloneReq
 }
+
+func updateReqWrapperForAnalytics(rw *openrtb_ext.RequestWrapper, adapterName string) {
+	reqExt, _ := rw.GetRequestExt() // TODO: Handle error
```

Comment:
For reviewers, I have a question about if I need to or how to handle an error here. I'm seeing if I can avoid having to pass the error upstream such that I have to return an error through `LogAuctionObject`, as it would cause me having to change the function signature of many other `LogObject` functions that also utilize `updateReqWrapperForAnalytics`. I'm thinking maybe if an error is returned from this call, we then just set `reqExtPrebid.Analytics = nil`

Reply:
I am leaning towards ignoring the error here. We have been discussing moving from the current lazy unmarshalling in the wrapper to a more aggressive mode. This would eliminate the possibility that `GetRequestExt()` could return an error, as it would already have been unmarshalled in the initial parsing step.